src/encord_active/frontend/src/components/explorer/Explorer.tsx

```diff
@@ -933,7 +933,11 @@ const GalleryItem = ({
   onShowSimilar: JSX.IntrinsicElements["button"]["onClick"];
   iou?: number;
 }) => {
-  const { data, isLoading } = useApi().fetchItem(itemId, iou);
+  const { data, isLoading } = useApi().fetchItem(
+    itemId,
+    iou !== undefined,
```

Comment:
We should check the scope instead, I think `iou` can be undefined

Reply:
@Encord-davids I fixed this now could you give it a look?